course_info/static/course_info/js/controllers/DetailsController.js

```diff
@@ -0,0 +1,110 @@
+// TODO - this is pretty much a copy of the people controller with some mods.
```

Comment:
We're going to clean this up as part of 2376? Or are these comments out-of-date / removable?